Wireframe/css/style.css

```diff
@@ -0,0 +1,56 @@
+.Title, footer{
+  text-align: center;
+}
+
+
+.first {
+  background-color: white;
+  width: 60%;
+  border: 15px solid rgb(199, 221, 199);
+  padding: 20px;
+  margin: auto;
+  border-radius: 20px;
+  margin-bottom: 30px;
+}
+
+.second, .third {
```

Comment:
If `.second` and `.third` have no difference. why not just use the one class?

Reply:
Hi @cjyuan, thank you for your reply. I have made several changes, please feel free to have a look.
1. first-child added
2. Header
3. Using link instead of button for "Read more"
4. Set width to css file
5. Same class for "second" and "third"

Thank you again for the comments, really appreciated!